cypress/integration/examples/viewport.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+context('Viewport', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/viewport')
+  })
+
+  it('cy.viewport() - set the viewport size and dimension', () => {
```

Comment:
The test case provided does not cover any of the required functionalities: login, add book, and delete book.